lib/monitoring/aws-rds/RdsClusterMetricFactory.ts

```diff
@@ -101,6 +101,14 @@ export class RdsClusterMetricFactory {
     return this.metric("CommitLatency", MetricStatistic.P90, "Commit");
   }
 
+  metricServerlessDatabaseCapacity() {
```

Comment:
Could potentially add a guard if this isn't applicable to non-`ServerlessCluster`s.

Reply:
Thinking guard would be along the lines of check if `this.cluster` is ServerlessCluster, but that would make the `cluster` attribute required not just a `clusterIdentifier`. I see note saying that `clusterIdentifier` is being deprecated, so forcing `cluster` in next revision would probably be a breaking change.

Reply:
Ah true. It could also just be best effort so we guard when we can (i.e. when `cluster` is available).